### PR TITLE
Receiver autocomplete

### PR DESCRIPTION
**Pull request by w0rm**
The receiver dropdown had too many results at SC and it was not possible to enter any arbitrary regex into the field. This pull request adds an autocomplete with a possibility to input any regex.

### CODE REVIEW

File: ui/app/src/Alerts/Types.elm [hunk 1]
```diff
@@ -18,17 +15,7 @@ type alias Alert =
     }
 
 
-type alias AlertGroup =
```
Comment (w0rm) [inline ui/app/src/Alerts/Types.elm]: `AlertGroup` and `Block` were not used anywhere

Comment (mxinden) [inline ui/app/src/Alerts/Types.elm]: Thanks for cleaning this up. This is a left over from switching to the /alerts endpoint. My bad.

File: ui/app/src/Views/AlertList/Views.elm [hunk 2]
```diff
@@ -91,6 +91,12 @@ alertGroups activeId filter groupBar alerts =
                         (alertList activeId labels filter)
                         (Dict.get labels grouped)
                 )
+            |> (\list ->
+                    if List.isEmpty list then
+                        [ Utils.Views.error "No alerts found" ]
```
Comment (mxinden) [inline ui/app/src/Views/AlertList/Views.elm]: 👍

File: ui/app/src/Utils/Match.elm [hunk 3]
```diff
@@ -1,4 +1,4 @@
-module Views.GroupBar.Match exposing (jaro, jaroWinkler, commonPrefix)
+module Utils.Match exposing (jaro, jaroWinkler, commonPrefix)
```
Comment (stuartnelson3) [inline ui/app/src/Utils/Match.elm]: great!

**Comment by w0rm** [comment]
@mxinden the dropdown only shows top 10 results, the same as in group by.

File: ui/app/src/Views/AlertList/Views.elm (hunk 2 above)
Comment (w0rm) [inline ui/app/src/Views/AlertList/Views.elm]: Yeah, we had this message in different place and it was never triggered. Now it is

**Comment by w0rm** [comment]
@mxinden feel free to create an issue to display receivers, but maybe we should hide this in the "info" collapsible section.

**Pull request merged**